tests/bazel_tests.bzl

```diff
@@ -169,32 +168,27 @@ def _bazel_test_script_impl(ctx):
     build_file = go.declare_file(go, path = "BUILD.in")
     ctx.actions.write(build_file, ctx.attr.build)
 
-    output = "external/" + ctx.workspace_name + "/" + ctx.label.package
-    targets = ["@" + ctx.workspace_name + "//" + ctx.label.package + t if t.startswith(":") else t for t in ctx.attr.targets]
+    output = "/".join([ctx.label.workspace_root if ctx.label.workspace_root else "external/{}".format(ctx.workspace_name), ctx.label.package])
```

Comment:
In general, I'd prefer to keep the string concatenation for paths where there are a known number of elements. I think `"/".join([...])` is less readable.